crates/voicevox_core_node_api/.cargo/config.toml

```
@@ -0,0 +1,5 @@
+[target.aarch64-unknown-linux-musl]
+linker = "aarch64-linux-musl-gcc"
+rustflags = ["-C", "target-feature=-crt-static"]
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-C", "target-feature=+crt-static"]
```

noeolなのがちょっと気になりました。

(このsuggestionが上手く動くかどうかはわかりませんが)

```suggestion
rustflags = ["-C", "target-feature=+crt-static"]

```